internal/client/consensus/grandpa/authorities.go

```diff
@@ -46,46 +49,64 @@ type hashNumber[H, N any] struct {
 }
 
 // appliedChanges represents the median and new set when a forced change has occurred
-type appliedChanges[H comparable, N constraints.Unsigned, ID AuthorityID] struct {
+type appliedChanges[H comparable, N constraints.Unsigned] struct {
 	median N
-	set    AuthoritySet[H, N, ID]
+	set    AuthoritySet[H, N]
+}
+
+// Get the current authorities and their weights (for the current set ID).
+func (sas *SharedAuthoritySet[H, N]) CurrentAuthorities() grandpa.VoterSet[string] {
```

Comment:
Since this is using currentAuthorities, perhaps rename it to something like `currentVoterSet` or `voterSetFromAuthorities`

Reply:
I used the same naming as substrate. I don't think I changed this in the move.